templates/default

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
I like the idea of this. I need to see what this flag exactly to make sure I'm not doing anything that might break other people's deployments. I also need to see if the right way to do is `export NODE_ENV="production"` and not just `NODE_ENV="production"`.

I actually don't write any `node` and wrote this tool as the guy who manages builds and deployments, so I'll have to do some research on this. Also, links or opinions are welcome.